README.md

```diff
@@ -306,6 +306,30 @@ After 5 seconds, input / output is restored back to normal.
 #### Other MITM arguments
 Run `pyrdp-mitm.py --help` for a full list of arguments.
 
+##### `--no-downgrade`
```

Comment:
Because of this new section the ToC will need to be regenerated. They used an online tool to do so. It kind of sucks that this is a manual step to be honest. Migrating this file to AsciiDoc would eliminate that problem. That said, just updating the ToC would be fine for now.